res/themes/light/icons/$icons.json

```diff
@@ -0,0 +1 @@
+{"icon":{"chat":{"value":"icons/chat.svg","type":"icon"},"check-circle":{"value":"icons/check-circle.svg","type":"icon"},"check":{"value":"icons/check.svg","type":"icon"},"chevron":{"value":"icons/chevron.svg","type":"icon"},"close":{"value":"icons/close.svg","type":"icon"},"computer":{"value":"icons/computer.svg","type":"icon"},"delete":{"value":"icons/delete.svg","type":"icon"},"error":{"value":"icons/error.svg","type":"icon"},"info":{"value":"icons/info.svg","type":"icon"},"lock":{"value":"icons/lock.svg","type":"icon"},"mobile":{"value":"icons/mobile.svg","type":"icon"},"thread":{"value":"icons/thread.svg","type":"icon"},"user":{"value":"icons/user.svg","type":"icon"},"visibility-invisible":{"value":"icons/visibility-invisible.svg","type":"icon"},"visibility-visible":{"value":"icons/visibility-visible.svg","type":"icon"},"web-browser":{"value":"icons/web-browser.svg","type":"icon"}}}
```

Comment:
What is this file for? What generates it? What reads it?

Reply:
This should not be included, I've explained it in this pull request description.

Reply:
Sorry I must be blind, I see explanation for why all the SVGs are included, but nothing about this file?

Reply:
It's just a copy paste of the output of what is in the design tokens. It won't be there when released

Reply:
What reads this file on this side of things?